pkg/controllers/chart/chart.go

```diff
@@ -53,7 +54,7 @@ const (
 var (
 	commaRE              = regexp.MustCompile(`\\*,`)
 	deletePolicy         = metav1.DeletePropagationForeground
-	DefaultJobImage      = "rancher/klipper-helm:v0.7.7-build20230403"
+	DefaultJobImage      = "brandond/klipper-helm:a5113bc"
```

Comment:
Will update once https://github.com/k3s-io/klipper-helm/pull/61 is merged and tagged